modules/tensor_mechanics/doc/content/modules/tensor_mechanics/Strains.md

```diff
@@ -95,6 +95,25 @@ In tensor mechanics, there are two decomposition options to obtain the strain in
 TaylorExpansion and EigenSolution, with the default set to TaylorExpansion. See the
 [ComputeFiniteStrain](/ComputeFiniteStrain.md) for a description of both decomposition options.
 
+### Large Strain Closed Loop Loading Cycle id=large_strain_closed_loop_loading_cycle
+
+A recent goal was to develop a consistent and more rigorous loading cycle for
+testing material models in regression tests. The closed loop loading cycle
+(i.e., extension, shear, unextension and unshear) shown below was used with
+finite strain and an elastic material. Since the initial and final
+configurations are the same, it was expected that the stress would return to
+the value of the undeformed state which was zero, however, this is not the case. 
```

Comment:
I'd lean towards writing something a bit more positive, like "As with other commercial FEM codes, a closed loop deformation path, such as the one shown in `[youd_have_to_give_the_figure_an_id_too]`, will not produce a stress value of zero when returning to the original configuration."  I'd also combine this one sentence with the paragraph below.

but, you know, whatever Ben writes ;-)

Reply:
I'd replace both of these paragraphs with this:

It is important to note that some inaccuracies are inherently present in incremental (hypoelastic) representations of  finite strain behavior. This is an issue for the MOOSE implementation discussed here, as well as the implementations of other major commercial codes. The magnitude of the error increases as the magnitude of the strain increases, so this is typically only a practical issue for very large strains, for which hyperelastic models are more appropriate. This can be demonstrated with the closed loop loading cycle illustrated in `[reference the figure]`. Since the initial and final configurations are the same, with elastic material, one would expect that the stress would return to zero in the final underformed state. With this incremental formulation, however, a residual stress remains, and the magnitude of that residual stress increases with the magnitude of the loading cycle.
This occurs because the integral of the rate-of-deformation over the full loading cycle is not zero.  A detailed explanation can be found in `[cite:belytschko2003]`.

Reply:
Done.